### ISSUE TITLE
multiple tune.run on root node does seem to mix scripts

### ISSUE THREAD

**Issue by thoglu** [ray-project/ray#18851]
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/ray/issues) and found no similar issues.

### Ray Component

Ray Tune

### What happened + What you expected to happen

I have a local cluster set up with one node being the root node via "ray start", and many other nodes connecting as worker node. I then start multiple "tune.run" with different scripts on the root node, sending each to the background via ctrl+z so the next "tune.run" can be started.

If i use the same script within each tune run, it will work fine and all the scripts get executed and I can train models on all the worker nodes in parallel.

If I use different scripts for each tune.run , tune will somehow mix run ids with different execution scripts, and I get error messages.

Is it even supposed to work that I execute multiple tune.run on the root node with different scripts in parallel?

### Reproduction script

Is this expected behavior? If this is not even supposed to work, we can skip a reproduction script.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by krfricke** [comment]
It is not recommended using multiple `tune.run` calls with the same cluster in parallel - here you should usually just enqueue your trials in a single Ray Tune run.

If you want to get rid of the errors you probably just need to set `TUNE_PLACEMENT_GROUP_CLEANUP_DISABLED=1` as an environment variable. You should do this for all runs, so

```
export TUNE_PLACEMENT_GROUP_CLEANUP_DISABLED=1
python run_tune.py
# ...
```

is probably what you're looking for.

**Comment by thoglu** [comment]
linking this to #21329

**Comment by krfricke** [comment]
Yeah, the `TUNE_PLACEMENT_GROUP_CLEANUP_DISABLED` should still work though it's untested and not a currently supported use case.
